cppa_slack_tracker/tests/test_workspace.py

```
@@ -0,0 +1,118 @@
+        root = get_workspace_root()
+        assert root == mock_get_workspace_path.return_value
+        assert "cppa_slack_tracker" in str(root)
+
+
+class TestGetRawRoot:
+    def test_raw_root_under_workspace_dir(self, mock_workspace_dir):
+        root = get_raw_root()
+        assert root == mock_workspace_dir / "raw" / "cppa_slack_tracker"
+        assert root.exists()
+
+    def test_raw_root_contains_raw_segment(self, mock_workspace_dir):
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_workspace_dir`

(ARG002)